src/blockchain_state_channel_v1.proto

```diff
@@ -32,29 +44,48 @@ message blockchain_state_channel_v1 {
 message blockchain_state_channel_response_v1 {
     bool accepted = 1;
     packet downlink = 4;
+    blockchain_state_channel_v1 active_sc = 5;
```

Comment:
do we even need this field?

Reply:
Hmm, I don't think we use it anywhere actually. I must have added it assuming we might. I'll remove it.